src/Analyzer/QueryPlanResult.php

```diff
@@ -6,12 +6,31 @@
 
 final class QueryPlanResult
 {
-    public const NO_INDEX = 'no-index';
-    public const TABLE_SCAN = 'table-scan';
-    public const UNINDEXED_READS = 'unindexed-reads';
+    public const ROW_NO_INDEX = 'no-index';
```

Comment:
renaming existing constants is a BC break. Please leave the old ones additionally in and mark them as deprecated